client/src/components/Charts/DayCounter.stories.jsx

```diff
@@ -0,0 +1,11 @@
+import DayCounter from './DayCounter';
+
+export default {
+  title: 'components/DayCounter',
+  component: DayCounter,
+};
+
+const Template = args => <DayCounter {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {};
```

Comment:
You don't have any `args` here. You can remove line 11 and `{...args}` in line 8. :)

Reply:
Done. Thank you.